### DISCUSSION TITLE
Event listeners beginning with show not work

### DISCUSSION THREAD

**Discussion by horstmue** [alpinejs/alpine#4275] [7. Bugs (archived)] [answered]
I discovered that event listener starting with "@show" will not work.

```
<div x-data="{open: false}"
	@show-select-price.window="alert('a')" >
   Test
</div>
```
Will not work

```
<div x-data="{open: false}"
	@sshow-select-price.window="alert('a')" >
    Test
</div>
```
will work correctly.

**Comment by SimoTod** [comment, marked as answer]
Can you post a reproducible example? It works correctly for me

**Reply by horstmue** [reply]
Thanks Simone,
You are absolutely right, and I am sorry not to test it before isolated.

![image](https://github.com/alpinejs/alpine/assets/13168829/524d2f69-3071-491c-9767-e916a8875d16)
The "@show" part is deleted during the Laravel compile process. 
You can see it in the image. 

I will update it here if I find out why this part was deleted.

**Comment by SimoTod** [comment]
The `@` symbol can be interpreted by blade (laravel templating engine) as
its own directive or it can confuse blade anyway. You can either escape it
by typing `@@show-...` (
https://laravel.com/docs/11.x/blade#blade-and-javascript-frameworks) or use
the extended syntax `x-on:show-...`

**Reply by horstmue** [reply]
This is the solution to my problem. Thank you so much.
